docs/encyclopedia/temporal-nexus-overview.mdx

```diff
@@ -0,0 +1,118 @@
+
+To make a Nexus Service available for others to use, a Nexus Endpoint must be created in the Nexus API Registry with a target Namespace and task-queue. 
+In pre-release, all Nexus requests will be sent to the single target Namespace and task-queue for the Endpoint. Future releases may support multiple Endpoint targets, similar to how an API proxy can have multiple targets.
+
+When you start a Nexus Operation in a caller Workflow, a command is sent to Temporal to schedule the Operation, and then the Temporal Nexus Machinery is responsible for making the Nexus RPC calls on your behalf. 
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with a Temporal Service.
+
+Similar to executing an Activity, for a Nexus Operation:
+
+1. Caller Workflow will schedule a Nexus Operation using the Temporal SDK, which sends a `ScheduleNexusOperation` command in the existing `RespondWorkflowTaskCompleted` RPC which is recorded as `NexusOperationScheduled` in the caller’s Workflow history.
+2. Nexus handler Worker uses the Temporal SDK to `PollNexusTaskQueue` and process the Nexus request.
+3. Nexus handler Worker sends a `RespondNexusTaskCompleted` via the SDK, with the result of the Nexus request, which may be one of: Started, Completed, Failed, TimedOut, Canceled,  see Nexus Operation Events for more info
+4. Caller Workflow then processes the Nexus Operation event via `PollWorkflowTaskQueue`.
+
+See also: [Nexus Architecture on GitHub](https://github.com/temporalio/temporal/blob/main/docs/architecture/nexus.md).
+
+## Key Capabilities
+
+Nexus support is available in Temporal Cloud, OSS Server, and Temporal SDKs:
+
+- Workers that handle Nexus Operations, listening on a Namespace and Task Queue
+- Nexus API registry, to create Nexus Endpoints that forward requests to a target Namespace and Task Queue, so others can use Nexus services via a Nexus endpoint.
+- Start Nexus operations from Temporal Workflows across team & Namespace boundaries
+- Observe and troubleshoot end-to-end execution
+
+Temporal Cloud provides enhanced security, observability, and multi-region routing:
+
+- Multi-region routing from Caller to destination Namespace in a Temporal Cloud account
+- Control who can manage a Nexus Endpoint with a new Nexus Endpoint Admin permission
+- Control who can use a Nexus Endpoint within an account, using Namespace allowlists
+- Observe Nexus audit logs and observability metrics via cloud integrations
+
+#### Integrated Temporal SDK experience
+
+- Run Nexus Services in a Temporal Worker:
+  - Abstract and expose a subset of Temporal primitives for others to use
+  - Can also execute arbitrary code
+  - Typically the same Worker as the underlying Temporal primitives
+  - Same queue-based Worker architecture, for Worker fleet load balancing
+  - No separate deployment for a bespoke service wrapper
+  - Polling architecture enables workers to run in a broad range of runtime environments
+- Use Nexus Services from a Workflow
```

Comment:
Is this a separate list? It doesn't seem like it belongs in the list above and doesn't have a colon.

Reply:
Also, seems redundant with the item below.